Bottom: dccf695dfe5e751cd6327fd3cc3cd126bd8538b7
Top:    a927ad5367c08f6e2136b5d0f9bcfac687b41675
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-13 22:27:52 +0000

Refresh of polygon-intersection-additions

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 671b7f8..1b0056c 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1077,10 +1077,10 @@ vertex_in_seg_rounded (const BoxType * b, void *cl)
   new_node = node_add_single_point (s->v, i->v->point);
   if (new_node != NULL)
     {
-//#ifdef DEBUG_INTERSECT
+#ifdef DEBUG_INTERSECT
       DEBUGP ("found new rounded intersection on segment \"s\" at (%$mn, %$mn)\n",
               i->v->point[0], i->v->point[1]);
-//#endif
+#endif
       i->node_insert_list =
         prepend_insert_node_task (i->node_insert_list, i->rtree_over_poly, s, new_node);
       s->intersected = 1;
